docs/tutorial.rst

```diff
@@ -187,6 +188,7 @@ Let's see how the method definition for pressing the search button would look.
 .. literalinclude:: code/round_2/pages.py
     :language: python
     :lines: 21-51
+    :emphasize-lines: 11-
```

Comment:
No code shows up in the preview https://github.com/billagee/bok-choy/blob/doc-edits/docs/tutorial.rst#define-the-search-method